lmdeploy/pytorch/models/q_modules.py

```diff
@@ -137,7 +150,8 @@ def forward(self, input):
         """
 
         if isinstance(input, torch.Tensor):
-            input_quant, input_scale = per_token_quant_int8(input, 1e-7)
+            input_quant, input_scale = per_token_quant_int8(
```

Comment:
In what case this branch will be called?

Reply:
this may happen when the pre op is a non-quant op and its output is feeding to the quant-linear